vm/devices/net/netvsp/src/lib.rs

```diff
@@ -1962,6 +1964,29 @@ fn read_packet_data<T: IntoBytes + FromBytes + Immutable + KnownLayout>(
     reader.read_plain().map_err(PacketError::Access)
 }
 
+fn parse_incoming_completion<'a, T: RingMem>(
+    completion: &CompletionPacket<'a, T>,
+) -> Result<PacketData, PacketError> {
+    if completion.transaction_id() == VF_ASSOCIATION_TRANSACTION_ID {
+        // Ideally netvsp could use the VF_ASSOCIATION completion packet as a signal
+        // that it is safe to expose the AccelNet device to the guest.
+        // Some Linux guests lack support for inline completions; they do not send the packet.
+        // Since the completion packet is not sent by all guests, it gets ignored.
+        Err(PacketError::IgnoreVfAssociationCompletion)
```

Comment:
I do not think it's a good idea to report expected behavior as an error that you then have to filter back out.